app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/JourneyStatus.scala

```diff
@@ -54,6 +54,12 @@ object JourneyStatus {
       subscriptionResponse: SubscriptionResponse
     ) extends SubscriptionStatus
 
+    // subscription has been submitted to ETMP
```

Comment:
could we move this outside of the `SubscriptionStatus` sealed trait and have it under `JourneyStatus` or some other trait? When we come to displaying the homepage it'd be nice to use this `JourneyStatus` as the state to look for before displaying it - `SubscriptionStatus` ties it to the subscription journey but we should be able to get to this state without going through the subscription journey every time

Reply:
:thinking: super hard to know what to call it - any ideas? :confounded:

Reply:
```scala
  sealed trait SubscriptionComplete extends JourneyStatus with Product with Serializable

  object SubscriptionComplete {
    // subscription has been submitted to ETMP
    final case class Subscribed(
      subscriptionDetails: SubscriptionDetails,
      subscriptionResponse: SubscriptionResponse
    ) extends SubscriptionStatus
  }
```